ERP-BACKEND/app/models/base.py

```
@@ -0,0 +1,31 @@
+    JSONB = JSON
+
+
+class TimestampMixin:
+    """Mixin for created_at and updated_at timestamps."""
+    created_at = Column(DateTime(timezone=True), server_default=func.now())
+    updated_at = Column(DateTime(timezone=True), server_default=func.now(), onupdate=func.now())
+
+
+class SoftDeleteMixin:
+    """Mixin for soft delete functionality."""
+    is_deleted = Column(Boolean, nullable=False, server_default="false")
```

Ruff (0.16.1)

[error] Undefined name `Boolean`

(F821)